sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilterSuite.scala

```diff
@@ -47,39 +49,47 @@ import org.apache.spark.util.{AccumulatorContext, AccumulatorV2}
  *    data type is nullable.
  */
 class ParquetFilterSuite extends QueryTest with ParquetTest with SharedSQLContext {
```

Comment:
can we just have a simple end-to-end test? The fix is actually very simple and seems not worth such complex tests to verify it.